src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorDiagnosticsTests.cs

```diff
@@ -402,5 +374,91 @@ public void ErrorTypeDeclarationWarns()
 
             CompilationHelper.AssertEqualDiagnosticMessages(expectedDiagnostics, result.Diagnostics);
         }
+
+        [Theory]
+        [InlineData(LanguageVersion.Default)]
+        [InlineData(LanguageVersion.Preview)]
+        [InlineData(LanguageVersion.Latest)]
```

Comment:
I'm curious if latest is always up to date... when 9 was released this would presumably be 9, presumably fine to ignore this though

Reply:
It should be, there's logic in each Roslyn release normalizing relative concepts like "latest" to an actual version number.